ONBOARD.md

```diff
@@ -0,0 +1,113 @@
+This document describes how to add new CI jobs (utilizing ci-operator) for
+OpenShift components to the OpenShift CI system. In general, two steps are
+needed:
+
+1. [Prepare configuration file](#prepare-configuration-for-component-repo),
```

Comment:
where is this config file supposed to be?

Reply:
openshift/release/ci-operator/...

Reply:
How about writing that here in the doc?

Reply:
Definitely! I will make the doc explicitly mention the "write config locally & test, then submit it to *there*" workflow and suggest the destination.